tests/benchmarks/garage/tf/algos/test_benchmark_ppo.py

```diff
@@ -45,7 +45,9 @@
     'n_epochs': 400,
     'max_path_length': 100,
     'batch_size': 2048,
-    'n_trials': 10
+    'n_trials': 10,
+    'optimize_minibatch_size': 32,
```

Comment:
i'm not sure this applies to our implementation since you are minibatching on paths and not steps.

Reply:
Are you talking about `optimize_minibatch_size`?
I'm grouping shuffled paths with size optimize_minibatch_size (in this case 32)
For example, if there are 100 paths, then there should be (32, 32, 32,  4) minibatch paths.

Reply:
yes -- what i'm saying is that this is not necessarily comparable with openai/baselines hyperparameters, which measure minibatches in *steps* and not *paths* -- so they will end up applying *far* more gradients than you will with the same hyperparameter.

Reply:
bump, has this been resolved @yonghyuc